packages/valory/skills/transaction_settlement_abci/behaviours.py

```diff
@@ -165,6 +180,165 @@ def get_gas_price_params(self, tx_body: dict) -> List[str]:
 
         return []
 
+    def _send_transaction_request(
```

Comment:
Why override it and not edit the original one?

https://github.com/valory-xyz/open-autonomy/blob/9af86efba420fe654e3eee9e304cc268c3470175/packages/valory/skills/abstract_round_abci/behaviour_utils.py#L1025